great_expectations/datasource/fluent/spark_dbfs_datasource.py

```diff
@@ -52,17 +57,27 @@ def add_csv_asset(
             header: boolean (default False) indicating whether or not first line of CSV file is header line
             infer_schema: boolean (default False) instructing Spark to attempt to infer schema of CSV file heuristically
             order_by: sorting directive via either list[Sorter] or "+/- key" syntax: +/- (a/de)scending; + default
+            batch_metadata: An arbitrary user defined dictionary with string keys which will get inherited by any
+                            batches created from the asset.
         """
         batching_regex_pattern: re.Pattern = self.parse_batching_regex_string(
             batching_regex=batching_regex
         )
         order_by_sorters: list[Sorter] = self.parse_order_by_sorters(order_by=order_by)
+        batch_metadata = (
```

Comment:
❤️ Interesting.  Seems fine for now; just wondering if this could foster supplying metadata in varying formats, which could be ultimately challenging to process consistently.